futures-util/src/stream/pending.rs

```diff
@@ -0,0 +1,29 @@
+use core::marker;
+use core::pin::Pin;
+
+use futures_core::{Stream, Poll};
+use futures_core::task;
+
+/// A stream which never returns any elements.
+///
+/// This stream can be created with the `stream::pending` function.
```

Comment:
```rust
/// A stream which never returns any elements.
///
/// This stream can be created with the `stream::pending` function.
```

nit: Could you change above doc-comment as follows? (because doc-comment of `pending` function gives a more detailed explanation. see also https://github.com/rust-lang-nursery/futures-rs/issues/1214)

```rust
/// Stream for the [`pending()`] function.
```